pages/s/[org]/[repo]/index.tsx

```diff
@@ -257,6 +262,22 @@ export default function RepoPage({ repoData, ogImageUrl }: RepoPageProps) {
                 </div>
               </div>
             </div>
+            <div className="flex w-fit max-w-xs lg:w-full lg:max-w-full gap-2 overflow-x-scroll lg:overflow-auto">
+              {repoData.language && (
+                <Link href={`/explore/topic/${getLanguageTopic(repoData.language)}/dashboard`}>
+                  <LanguagePill language={repoData.language.toLowerCase()} />
+                </Link>
+              )}
+              {repoData.license && (
+                <Pill text={repoData.license} icon={<FaBalanceScale />} size="xsmall" className="whitespace-nowrap" />
+              )}
+              <Pill
+                text={`Last Updated: ${new Date(repoData.updated_at).toLocaleDateString()}`}
```

Comment:
I think we should use the `last_pushed_at` date here instead of `updated_at`. The `updated_at` is tied to how frequently our ETL updates the repository, and could be out of date compared to the calculated `last_pushed_at` date, which comes from our Timescale data.

cc: @jpmcb

Reply:
Correct!